entrypoint.sh

```diff
@@ -170,21 +170,23 @@ insert into sym_router (router_id,
         source_node_group_id, target_node_group_id, create_time,
         last_update_time) values ('${GROUP_ID}-2-${REPLICATE_TO}','${GROUP_ID}', '${REPLICATE_TO}',
         current_timestamp, current_timestamp);
+EOL
 
-insert into sym_channel (channel_id, processing_order, max_batch_size, max_batch_to_send,
-         extract_period_millis, batch_algorithm, enabled)
-     values ('${REPLICATE_TABLE}', 10, 1000, 10, 0, 'default', 1);
+for REPLICATE_TABLE in $REPLICATE_TABLES
+do
+echo "Adding config for $REPLICATE_TABLE in ${DB_TYPE}..."
+echo -e "insert into sym_channel (channel_id, processing_order, max_batch_size, max_batch_to_send,
```

Comment:
Can we do this with cat and heredoc like we do above? (It screws up the indenting but I still think it's nicer.)